src/types/types.ts

```diff
@@ -100,6 +100,28 @@ export type ImageToolData<Actions = {}, AdditionalFileData = {}> = {
   } & AdditionalFileData;
 } & (Actions extends Record<string, boolean> ? Actions : {});
 
+/**
+ * @description Enable or disable features.
+ */
+export type FeaturesConfig = {
+  /**
+   * Flag to enable/disable tune - background.
+   */
+  background: boolean;
+  /**
+   * Flag to enable/disable tune - border.
+   */
+  border: boolean;
+  /**
+   * Flag to enable/disable caption.
+   */
```

Comment:
describe in jsdoc what will happen if boolean passed, and "optional"

Reply:
```
  /**
   * Flag to enable/disable caption.
   * Can be set to 'optional' to allow users to toggle via block tunes.
   */
  caption: boolean | 'optional';
  ```
  
  is this okay ?

Reply:
yes